CcWorks/Workers/Solvers/BrpMagicStringsSolver.cs

```diff
@@ -168,6 +171,18 @@ private static List<string> AnalyzeMagicStrings(SyntaxNode node)
             return result;
         }
 
+        private static string GetConstantName(string constantSubstring)
```

Comment:
This function should also:
- trim leading numbers
- truncate long names to (for example) 30 chars
- check if result name is empty of whitespace, then return default unique name